lms/djangoapps/course_api/serializers.py

```diff
@@ -33,9 +35,9 @@ class _CourseApiMediaCollectionSerializer(serializers.Serializer):  # pylint: di
     course_video = _MediaSerializer(source='*', uri_attribute='course_video_url')
 
 
-class CourseSerializer(serializers.Serializer):  # pylint: disable=abstract-method
+class CourseListSerializer(serializers.Serializer):  # pylint: disable=abstract-method
```

Comment:
Is CourseList a term we use elsewhere?  This makes me think this is a List.  Would CourseOverviewSerializer be more  accurate?  Is that the likely breaking point for course related details that would go in here vs in CourseDetailsSerializer?